### ISSUE TITLE
defaultmodify() fails when processing arrays

### ISSUE THREAD

**Issue by simoami** [rjrodger/seneca-web#18]
The [defaultmodify()](https://github.com/rjrodger/seneca-web/blob/master/web.js#L846) function doesn't properly test for objects since `_.isObject()` returns true for arrays as well. So, the following code fails as keys are numeric and k.indexOf() errors out:
web.js (line 846)

```js
  if( _.isObject( result.out ) ) {
    _.each(result.out,function(v,k){
      if(~k.indexOf('$') && 'http$' !== k) {
        delete result.out[k];
      }
    });
  }
```
Proof: http://jsbin.com/sokufoloqo/edit?js,console

My suggestion is to modify the condition to `_.isObject( result.out ) && !_.isArray( result.out )`

**Comment by maxired** [comment]
This is fixed by 7708df89c6fb8f2bda509b0ec3311fc22bb509be

@simoami , please, can you close this issue ?

**Comment by simoami** [comment]
works fine now, thanks!
